config.toml

```diff
@@ -101,6 +101,15 @@ manager_type = "OnDisk"
 
 # Example of an Mbed Crypto provider configuration.
 [[provider]]
+# ⚠
+# ⚠ WARNING: Provider name cannot change.
```

Comment:
I'd be inclined to specify why it shouldn't be changed, that it leads to existing keys being lost.